ctera/src/test/java/ch/cyberduck/core/ctera/CteraAttributesFinderFeatureTest.java

```diff
@@ -43,24 +40,24 @@ public class CteraAttributesFinderFeatureTest extends AbstractCteraTest {
     @Test(expected = NotfoundException.class)
     public void testFindNotFound() throws Exception {
         final Path test = new Path(new DefaultHomeFinderService(session).find(), new AlphanumericRandomStringService().random(), EnumSet.of(Path.Type.file));
-        final DAVAttributesFinderFeature f = new DAVAttributesFinderFeature(session);
+        final DAVAttributesFinderFeature f = new CteraAttributesFinderFeature(session);
         f.find(test);
         fail();
     }
 
     @Test
     public void testFindFile() throws Exception {
```

Comment:
Add test to show `guid` is same in attributes returned from touch and mkdir and when looking up with attributes finder.

Reply:
@dkocher this is an integration test, right? (As the `guid` comes from the portal).

Reply:
Yes, as in `CteraAttributesFinderTest`

Reply:
d00b676de99ac4571a563b1e4cfbc5a4f6c4c182